### PR TITLE
RET 1944 Session Timeout 

### PR DESCRIPTION
**Pull request by brendanmchugh**
### JIRA link

https://tools.hmcts.net/jira/browse/RET-1944

### Change description

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by marcjudge — APPROVED** [review]

**Review by lvo700 — DISMISSED** [review]

**Review by JustMehmet — DISMISSED** [review]

File: src/test/unit/assets/js/session/session-timeout.test.ts [hunk 1]
```diff
@@ -0,0 +1,305 @@
...
+    sandbox.restore();
+  });
+
+  describe('init', () => {
+    it('should call addListeners and extend session', () => {
+      addListenersStub = sandbox.stub(sessionTimeout, 'addListeners');
+      extendSessionStub = sandbox.stub(sessionTimeout, 'extendSession');
+      sessionTimeout.init();
+      expect(addListenersStub.calledOnce);
+      expect(extendSessionStub.calledOnce);
```
Comment (JustMehmet) [inline src/test/unit/assets/js/session/session-timeout.test.ts:72]: These expect checks does not assert anything gives code smell error

File: src/test/unit/assets/js/session/session-timeout.test.ts [hunk 2]
```diff
@@ -0,0 +1,305 @@
...
+      const extendButton = document.querySelector('#extend-session');
+      sessionTimeout.addListeners();
+      const evt = new MouseEvent('click', {
+        view: window,
+        bubbles: true,
+        cancelable: true,
+        clientX: 20,
+      });
+      extendButton.dispatchEvent(evt);
+      expect(extendSessionStub.calledOnce);
```
Comment (JustMehmet) [inline src/test/unit/assets/js/session/session-timeout.test.ts:88]: These expect checks does not assert anything gives code smell problem in sonar cloud

File: src/test/unit/assets/js/session/session-timeout.test.ts [hunk 3]
```diff
@@ -0,0 +1,305 @@
...
+      const extendButton = document.querySelector('#extend-session');
+      sessionTimeout.removeListeners();
+      const evt = new MouseEvent('click', {
+        view: window,
+        bubbles: true,
+        cancelable: true,
+        clientX: 20,
+      });
+      extendButton.dispatchEvent(evt);
+      expect(extendSessionStub.notCalled);
```
Comment (JustMehmet) [inline src/test/unit/assets/js/session/session-timeout.test.ts:104]: These expect checks does not assert anything gives code smell problem in sonar cloud

File: src/test/unit/assets/js/session/session-timeout.test.ts (hunk 1 above)
Comment (brendanmchugh) [reply src/test/unit/assets/js/session/session-timeout.test.ts]: Thanks Mehmet, those incorrect expect checks have been updated.

**Review by marcjudge — DISMISSED** [review]

**Review by lvo700 — DISMISSED** [review]

**Review by marcjudge — APPROVED** [review]

**Review by DarylBoam — APPROVED** [review]

**Pull request merged**